### ISSUE TITLE
WebNowPlaying-issues with duration and position

### ISSUE THREAD

**Issue by MZimmermann** [khanhas/Spicetify#50]
Hi,

I'm trying to use the rainmeter-WebNowPlaying-plugin with spicetify and had an issue getting proper duration- and positions. It worked with the browser-extension but not with spicetify. Since other information was transmitted just fine, it wasn't a connection issue. Turns out, that the rainmeter-plugin looks for the InfoTypes "position" and "duration" and expects those to be in the format MM:SS.
But the wnpPlugin-script in spicetify transmits this information under the field-names "positionString" and "durationString" (which aren't evaluated at all). It worked, after I tentatively added the lines
```
musicInfo.position = musicInfo.positionString;
musicInfo.duration = musicInfo.durationString;
```
into the setup-method, thus using the same lambdas to fill both values. But honestly it looks to me as if the *String-versions of those fields might not be needed at all, since the plugin doesn't process them.

After that, it nearly worked. Unfortunately, the convertTimeToString doesn't truncate the timeInSeconds-parameter. The rainmeter-plugin insists of it being an integer, and if the string is transmitted something like "3:26.6666667", it can't be handled.

On the whole I'm not really a fan of transmitting durations as formatted Strings with time-units, milliseconds as integers would be much less error-prone. But in this case it seems to be the rainmeter-plugin defining the format, so Spicetify should probably just send what's expected.

**Comment by khanhas** [comment]
I just checked WebNowPlaying plugin source. It only accept formatted duration and position string.  
If I send only seconds value, string value of Duration and Position measure in Rainmeter are not formatted.
And you're right, I sent them with wrong prefixes the whole time.
Fix is up in [1.7.3](https://github.com/khanhas/Spicetify/releases/tag/1.7.3), check it out.

**Comment by MZimmermann** [comment]
Thank you, now everything works out-of-the-box :)
